driver/ppm_fillers.c

```diff
@@ -322,10 +325,23 @@ int f_sys_open_x(struct event_filler_arguments *args)
 	if (unlikely(res != PPM_SUCCESS))
 		return res;
 
+	if (unlikely(!get_fd_dev_ino(retval, &dev, &ino)))
+	{
+		dev = 0;
+		ino = 0;
+	}
```

Comment:
We can avoid all this stuff, initializing at the beginning `dev` and `ino` as we do in BPF